robot/basestation/static/js/elements/tools/error_handling.js

```diff
@@ -0,0 +1,18 @@
+function flaskError (jqXHR, exception) {
```

Comment:
I would suggest you slightly expand this function with a third argument on the URL of the error.

Suppose we have multiple systems that use your flaskError function then if we see "error 404" we don't actually know what subsystem/action triggered it. 

So I would like to see a message like "Error 404 when accessing localhost:5000/rover" or something